queries/nvim/folds.scm

```diff
@@ -0,0 +1,13 @@
+[
+  (begin_block)
+  (block)
+  (case)
+  (class_def)
```

Comment:
Should include fun_def (or at least top level ones, somehow), lib_def, struct_def, enum_def, maybe some others I'm forgetting

Reply:
Added a lot that was missing now, thanks!